api/src/controller/http/dev/dev.v1.controller.ts

```diff
@@ -23,4 +31,13 @@ export class DevV1Controller {
   ) => {
     res.send({ message: 'Hello World!' });
   };
+
+  public serverTime = async (
+    req: Request,
+    res: Response<ServerTimeV1Response>
+  ) => {
+    const time = Date();
+
+    res.send({ message: time });
```

Comment:
조금더 여러 언어와 호환성이 좋은 ISO8601 포맷으로 시간을 리턴해주면 좋을 것 같습니다.
```suggestion
    const now = new Date();

    res.send({ message: now.toISOString() });
```

Reply:
[3992fda](https://github.com/MovieReviewComment/Mr.C/pull/76/commits/3992fda7dd083fc9e2d0c6adedcd1eeb07b0cedf) 에서 반영했습니다